README.md

```diff
@@ -3,3 +3,15 @@
 
 You have a cool idea for an algorithms project? Use this repository. Make sure
 to explain what problem you're solving and how you're doing it.
+
+## Kruskal's Algorithm
+
+Implement Kruskal's algorithm, and find the worst-case time complexity.
+
+## Runtime Analysis
+
+In the worst case, this algorithm will search through every edge and have bad cases with the detecting cycles function.  At the beginning of my program, it creates an empty tree of size $|V|$.  Then, it builds a priority-queue-type-thing that traverses every edge, so it takes $|E|$ time.  Next, it runs through a while loop that iterates over every edge from the queue ($|E|$).  Inside this loop, it takes the next item out of the priority queue, which could at most take $|H|$ time, where $H$ is the longest edge.  After that, it just does some linear stuff and also runs the hasCycle() funciton that I wrote for a different project, which had a time complexity of $|E| + |V|$.
```

Comment:
Don't understand how the length of an edge matters -- and what do you mean by that? Each edge is just one edge, right?